addons/third-party-button/addon.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "Third-party button",
+  "description": "Adds buttons for viewing projects on forkphorus or other sites.",
+  "userscripts": [
+    {
+      "url": "userscript.js",
+      "matches": ["https://scratch.mit.edu/projects/*"]
+    }
+  ],
+  "options": [

```

Comment:
It's cool that you used options, but are they really necessary?

Reply:
I think that having them is neat so I really dunno why not. The "line sprite effect" as I call it: A feature that isn't really useful, but it would just be efford to remove it so why bother.